Meta/Lagom/Fuzzers/FuzzGzipRoundtrip.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCompress/Gzip.h>
+#include <stdio.h>
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* data, size_t size)
```

Comment:
Please rebase onto the latest `master`, all the fuzzers now have an additional call to `AK::set_debug_enabled(false)` to avoid printing debug output that slows down performance.